GenericSerialDeviceRemoteControl/GenericSerialDeviceRemoteControl.py

```diff
@@ -0,0 +1,350 @@
+from __main__ import vtk, slicer
+from slicer.ScriptedLoadableModule import ScriptedLoadableModule, ScriptedLoadableModuleWidget, ScriptedLoadableModuleLogic
+from slicer.util import VTKObservationMixin
+import logging
+
+
+class GenericSerialDeviceRemoteControl(ScriptedLoadableModule):
+    def __init__(self, parent):
+        ScriptedLoadableModule.__init__(self, parent)
+        self.parent.title = "Generic Serial Device Remote Control"
+        self.parent.categories = ["IGT"]
+        self.parent.dependencies = ["OpenIGTLinkRemote"]
+        self.parent.contributors = ["Adam Aji (SonoVol, PerkinElmer)"]
+        self.parent.helpText = """
+            Send commands to and read output from serial devices over their CTS/RTS/TX lines.
```

Comment:
Mention Plus toolkit and provide a link to a documentation page with some more details.